packages/web-twig/src/Resources/components/Header/stories/HeaderInvertedWithActionsAndDropdown.twig

```diff
@@ -0,0 +1,92 @@
+<Header color="inverted">
+    {% include '@components/Header/stories/_partials/SpiritLogo.twig' %}
+
+    <HeaderMobileActions dialogId="header_dialog_example_3_main_navigation">
+        <DropdownWrapper>
+            <Button
+                color="inverted"
+                data-spirit-toggle="dropdown"
+                data-spirit-target="#dropdownUserMobile"
+                isSquare
+            >
+                <Icon name="profile" />
+                <VisuallyHidden>My Profile</VisuallyHidden>
+            </Button>
+            <Dropdown id="dropdownUserMobile" placement="bottom-right">
+                <Stack hasSpacing>
+                    <a href="#">Dashboard</a>
+                    <a href="#">Profile</a>
+                    <a href="#">Favourites</a>
+                    <a href="#">Notifications</a>
+                    <a href="#">Sign out</a>
+                </Stack>
+            </Dropdown>
+        </DropdownWrapper>
+    </HeaderMobileActions>
+
+    <HeaderDesktopActions aria-label="Main navigation">
+        <HeaderNav>
+            <HeaderNavItem>
+                <HeaderLink href="#" isCurrent>Job offers</HeaderLink>
+            </HeaderNavItem>
+            <HeaderNavItem>
+                <HeaderLink href="#">Part-time jobs</HeaderLink>
+            </HeaderNavItem>
+            <HeaderNavItem>
+                <HeaderLink href="#">Inspiration</HeaderLink>
+            </HeaderNavItem>
+            <HeaderNavItem>
+                <HeaderLink href="#">Replies</HeaderLink>
+            </HeaderNavItem>
+            <HeaderNavItem>
+                <HeaderLink href="#">Employers</HeaderLink>
+            </HeaderNavItem>
+        </HeaderNav>
+    </HeaderDesktopActions>
+
+    <HeaderDesktopActions color="secondary">
+        <HeaderNav>
+            <HeaderNavItem UNSAFE_style="position: relative;">
+                <HeaderLink
+                    href="#"
```

Comment:
⚠️ This `HeaderLink` modifies the URL and thus if the page is scrolled down, moves the page up. I think some kind of button should be used instead.

Reply:
I think it should be enough to use `HeaderButton` instead.